src/hooks/useUpdateAcceptedTerms.ts

```diff
@@ -0,0 +1,48 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useAccount } from 'wagmi';
+import { requestGraphQL } from '@/helpers/request';
+import { USER_ACCEPT_TERMS_MUTATION } from '@/queries/user.query';
+
+export const useUpdateAcceptedTerms = () => {
+  const { address } = useAccount();
+  const queryClient = useQueryClient();
+
+  return useMutation({
+    mutationKey: ['user', address, 'acceptedToS'],
+    mutationFn: async (acceptedToS: boolean) => {
+      return await requestGraphQL(
+        USER_ACCEPT_TERMS_MUTATION,
+        {
+          acceptedToS,
```

Comment:
It doesn't accept any parameter!
There is only one way to call it, and that set that value true